config.json.sample

```diff
@@ -1,18 +1,6 @@
 {
 	"backendUrl": "http://localhost/owncloud",
-	"apps": {
```

Comment:
why drop apps? why own apps.json? One config to rule them all - right?

You wanted to reduce requests ;-)